wicket-jquery-ui-parent/wicket-kendo-ui/pom.xml

```diff
@@ -13,8 +13,8 @@
 	<artifactId>wicket-kendo-ui</artifactId>
 	<packaging>bundle</packaging>
 
-    <properties>
-	    <bundle.namespace>com.googlecode.wicket.kendo.ui</bundle.namespace>
+	<properties>
+		<bundle.namespace>com.googlecode.wicket.kendo.ui</bundle.namespace>
```

Comment:
This change seems unrelated. I'm also a bit confused because the master branch says `org.wicketstuff.kendo.ui` here.